config

```diff
@@ -0,0 +1 @@
+github/ci/prow-deploy/files/
```

Comment:
I think that we should flip the order and move the real content to `config` and linke the other places. the same for `jobs`.